components/fibery/actions/get-entity-or-create/get-entity-or-create.mjs

```diff
@@ -0,0 +1,16 @@
+import common from "../common/common-entities.mjs";
+
+export default {
+  ...common,
+  key: "fibery-get-entity-or-create",
+  name: "Get or Create Entity",
+  description: "Get an entity or create one if it doesn't exist. [See the docs here](https://api.fibery.io/graphql.html#create)",
+  version: "0.0.1",
+  type: "action",
+  async run({ $ }) {
+    const entities = await this.findEntities($);
+    return entities.length
```

Comment:
Perhaps this could export a summary based on which course of action was taken

```suggestion
    const { length } = entities;
    const summary = length ? "summary" : "otherSummary";
    $.export("$summary", summary);
    return length
```